```
[ 61%] Building C object tests/CMakeFiles/mariadb-client-test.dir/mysql_client_test.c.o
 In file included from /home/builder/.termux-build/mariadb/src/tests/mysql_client_test.c:38:
 /home/builder/.termux-build/mariadb/src/tests/mysql_client_fw.c:1438:3: error: 'compile_time_assert' declared as an array with a negative size
   compile_time_assert(sizeof(MYSQL) == 77*sizeof(void*)+656);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 /home/builder/.termux-build/mariadb/src/include/my_global.h:384:38: note: expanded from macro 'compile_time_assert'
     typedef char compile_time_assert[(X) ? 1 : -1] __attribute__((unused)); \
                                      ^~~~~~~~~~~~
 1 error generated.
```

No idea why only 32bit Arm need this for compile to be successful

--- a/tests/mysql_client_fw.c
+++ b/tests/mysql_client_fw.c
@@ -1430,7 +1430,7 @@
     tests_to_run[i]= NULL;
   }
 
-#ifdef _WIN32
+#if defined(_WIN32) || (defined(__ANDROID__) && defined(__arm__))
   /* must be the same in C/C and embedded, 1208 on 64bit, 968 on 32bit */
   compile_time_assert(sizeof(MYSQL) == 60*sizeof(void*)+728);
 #else
